js/lykke.js

```diff
@@ -401,13 +401,17 @@ module.exports = class lykke extends Exchange {
     }
 
     parseTicker (ticker, market = undefined) {
+        // {
+        //     "assetPair":"ADAUSD",
+        //     "volume24H":264.6398,
+        //     "lastPrice":1.29535,
+        //     "bid":1.28805,
+        //     "ask":1.29074
+        // }
         const timestamp = this.milliseconds ();
-        let symbol = undefined;
-        if (market) {
-            symbol = market['symbol'];
-        }
+        const symbol = this.safeSymbol (undefined, market);
```

Comment:
Let's take `assetPair` into account here, if possible. That is the whole point of adding these responses – to see what kind of structure we're actually dealing with and to adjust the code accordingly.